libs/payments/cart/.eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
+  "extends": ["../../../.eslintrc.json"],
+  "ignorePatterns": ["!**/*"],
```

Comment:
Ha, I actually don't know. This is a default param from the Nx generator.

Reply:
We might want to confirm that. I didn't clone+lint here, but I quickly threw it into the Monitor repo's .eslintrc.json since I had that project handy and it started linting a bunch of files in my ./node_modules/** folder which were previously ignored (because, like, /node_modules/).

Reply:
Ah OK, I found [this SO post](https://stackoverflow.com/questions/71886802/how-to-ignore-specific-files-folders-from-eslint-within-nx-workspaces-nrwl) that helped me make sense of things.

In a typical NX install, it seems Nx will have an `ignorePattern` of `["**/*"]` to ignore everything, and then for each lib or app, it'll have an `ignorePattern` of `["!**/*"]` to un-ignore everything. Regarding `node_modules` this should be safe since in the integrated pattern of NX there's only a root level `node_modules`, so there won't be ones in the individual libs or apps folders.

Regarding the root level `node_modules` I believe it's being ignored by plugin `@typescript-eslint`.